python/src/eloqstore/client.py

```
@@ -0,0 +1,339 @@
+
+    def __exit__(self, exc_type, exc, tb) -> None:
+        self.close()
+
+    def put(
+        self,
+        key: str | bytes,
+        value: str | bytes | bytearray | memoryview | Any,
+        *,
+        timestamp: int | None = None,
+    ) -> None:
+        key_arr, key_ptr, key_len = alloc_bytes(_to_bytes(key))
```

Ruff (0.15.12)

[warning] Unpacked variable `key_arr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)